tests/static/js/jqunit/jqUnit-binder-commonEventBindings.js

```diff
@@ -0,0 +1,158 @@
+/* globals fluid, jqUnit */
+(function () {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+
+    // Component to test support for common event bindings
+    fluid.defaults("gpii.tests.binder.commonEventBindings", {
+        gradeNames: ["gpii.tests.binder.base", "gpii.binder.bindMarkupEvents"],
+        bindings: {
+        },
+        selectors: {
+            inputButton: "#input-button",
+            inputKeydown: "#input-button",
+            paragraphClick: ".paragraph-click",
+            multipleEventInputButton: "#input-multiple-events-button",
+            multipleEventHandlerInputButton: "#input-multiple-handlers-events-button"
+        },
+        markupEventBindings: {
+            inputButton: {
+                method: "click",
+                args: ["{that}.handleInputClick"]
+            },
+            inputKeydown: {
+                method: "keydown",
+                args: ["{that}.handleInputClick"]
+            },
+            paragraphClick: {
+                method: "click",
+                args: ["gpii.tests.binder.commonEventBindings.handleParagraphClick({arguments}.0)"]
+            },
+            multipleEventInputButton: {
+                method: ["click", "keydown"],
+                args: ["{that}.handleInputClick"]
+            },
+            multipleEventHandlerInputButton: [
+                {
+                    method: "click",
+                    args: ["{that}.handleInputClick"]
+                },
+                {
+                    method: "keydown",
+                    args: ["{that}.handleInputKeydown"]
+                }
+            ]
+        },
+        invokers: {
+            handleInputClick: {
+                funcName: "gpii.tests.binder.commonEventBindings.handleInputClick",
+                args: ["{arguments}.0"]
+            },
+            handleInputKeydown: {
+                funcName: "gpii.tests.binder.commonEventBindings.handleInputKeydown",
+                args: ["{arguments}.0"]
+            }
+        },
+        listeners: {
+            "onCreate.markupEventBindings": "{that}.events.onMarkupRendered.fire()"
```

Comment:
This is the kind of thing I'd also write out, what if someone needs to do other work before firing the event?  I guess they can just rewrite the whole string.